### PR TITLE
Refactor admin/overview to use Composition API

### PR DESCRIPTION
**Pull request by pixeldesu**
# What

Use Composition API for `admin/overview` page

# Why

Refactor to Composition API (per contribution guide)

### CODE REVIEW

File: packages/client/src/pages/admin/overview.vue [hunk 1]
```diff
@@ -65,103 +65,73 @@
...
-	},
-
-	async mounted() {
-		os.api('meta', { detail: true }).then(meta => {
-			this.meta = meta;
+import { instance } from '@/instance';
+import { i18n } from '@/i18n';
+
+let stats: any = $ref(null);
+let meta: any = $ref(instance);
```
Comment (Johann150) [inline packages/client/src/pages/admin/overview.vue]: `meta` is not used, so seems unnecessary.

File: packages/client/src/pages/admin/overview.vue [hunk 2]
```diff
@@ -65,103 +65,73 @@
...
+	if (typeof q === 'string') {
+		instance = await os.api('federation/show-instance', {
+			host: q
 		});
-		
-		os.api('stats', {}).then(stats => {
-			this.stats = stats;
+	}
+	// TODO
+}
```
Comment (Johann150) [inline packages/client/src/pages/admin/overview.vue]: I think `showInstanceInfo` is also unused?

File: packages/client/src/pages/admin/overview.vue [hunk 3]
```diff
@@ -65,103 +65,73 @@
...
+			notesComparedToThePrevDay = stats.originalNotesCount - chart.local.total[1];
 		});
+	});
 
-		this.$nextTick(() => {
-			this.queueStatsConnection.send('requestLog', {
-				id: Math.random().toString().substr(2, 8),
-				length: 200
-			});
+	os.api('admin/server-info', {}).then(serverInfoResponse => {
```
Comment (Johann150) [inline packages/client/src/pages/admin/overview.vue]:
```suggestion
	os.api('admin/server-info').then(serverInfoResponse => {
```

**Review by Johann150 — APPROVED** [review]

**Comment by syuilo** [comment]
🙏

**Pull request merged**